pl/README.md

```diff
@@ -0,0 +1,39 @@
+# Uwagi dotyczące tłumaczenia
+
+Celem tego pliku jest zdefiniowanie pewnych wytycznych dla ludzi, którzy chcą uczestniczyć w pracach nad tłumaczeniem tego projektu. Jak wiadomo, tłumaczenie z jednego języka na drugi bywa niezwykle złożonym zadaniem, przepełnionym elementami subiektywnego podejścia do tematu. Z tego też powodu wydaje się być niezbędnym, by wyznaczać pewne zalecenia, które będą powszechnie uznane za obowiązujące każdego, kto podejmuje się tłumaczenia. Przykładem takich reguł mogą być zasady dotyczące tłumaczenia określonych wyrażeń, bądź wyznaczenie takich wyrażeń, których z uwagi na ich kontekst czy znaczenie nie powinno się tłumaczyć.
+
+## Zalecenia
+
+Obecnie brak
```

Comment:
That's not valid in my opinion. People doing translation to Polish tend to use masculine verb version (for a variety of different reasons). So the guideline could just kindly suggest to not write a translation like this: `Gdzie *umieściłeś* swoje pliki`, 'Gdzie mogą znaleźć je *koledzy* z zespołu` with a masculine verb conjugation but instead use more neutral language: 'Gdzie zamieszczone zostały pliki`, 'Gdzie mogą je znaleźć inni z zespołu', etc. You know, to now let poeple recognize at the first look that translation was made by guys for other guys.
Sometimes I'm doing such small contributions to other projects to correct wording in that way.